### DISCUSSION TITLE
SPARQL EXISTS

### DISCUSSION THREAD

**Discussion by mgberg** [ontop/ontop#511] [Q&A] [answered]
I see in the documentation that `EXISTS` / `NOT EXISTS` is currently not supported. Are there any plans to add support for it anytime soon?

**Comment by bcogrel** [comment, marked as answer]
There is no concrete plan yet but that's something we would like to have. For that, we would have to introduce support for top-down evaluation in our internal algebra, I foresee that this will be a bit of work.

In the meantime, MINUS can be used as an (imperfect) workaround.
